README.md

```diff
@@ -53,52 +53,86 @@ wget https://github.com/sealerio/sealer/releases/download/v0.9.0/sealer-v0.9.0-l
 tar zxvf sealer-v0.9.0-linux-amd64.tar.gz && mv sealer /usr/bin
 ```
 
-Build a ClusterImage with Kubernetes dashboard:
+## Install a kubernetes cluster
 
-Kubefile:
+```shell
+# run a kubernetes cluster
+sealer run docker.io/sealerio/kubernetes:v1.22.15 \
+  --masters 192.168.0.2,192.168.0.3,192.168.0.4 \
+  --nodes 192.168.0.5,192.168.0.6,192.168.0.7 --passwd xxx
+```
 
-```shell script
-# base ClusterImage contains all the files that run a kubernetes cluster needed.
-#    1. kubernetes components like kubectl kubeadm kubelet and apiserver images ...
-#    2. docker engine, and a private registry
-#    3. config files, yaml, static files, scripts ...
-FROM registry.cn-qingdao.aliyuncs.com/sealer-io/kubernetes:v1.22.15
-# download kubernetes dashboard yaml file
-RUN wget https://raw.githubusercontent.com/kubernetes/dashboard/v2.2.0/aio/deploy/recommended.yaml
-# when run this ClusterImage, will apply a dashboard manifests
-CMD kubectl apply -f recommended.yaml
+## Build an app image
+
+nginx.yaml:
+
+```shell
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: my-nginx
+  namespace: default
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      run: my-nginx
+  template:
+    metadata:
+      labels:
+        run: my-nginx
+    spec:
+      containers:
+        - name: my-nginx
+          image: nginx
```

Comment:
use full path : docker.io/library/nginx:latest

Reply:
> 

@kakaZhou719  This is defined in yaml and does not need to be modified